commands_for_exercises.txt

```diff
@@ -0,0 +1,63 @@
+##Logging to Discovery:
+ssh -Y <yourusername>@login.discovery.neu.edu
+
+##Copying training material to Discovery using 'scp':
+git clone <url>
+
+##Exercise 1:
+# During training:
+srun --partition=reservation --reservation=bootcamp_gpu_2022 --nodes=1 --cpus-per-task=4 --gres=gpu:1 --mem=10G --time=02:00:00 --pty /bin/bash
```

Comment:
--reservation=bootcamp_gpu_2022 -> --reservation=bootcamp_gpu_2023

Reply:
I have now updated this @joseph-cooke